lift/Readme.md

```diff
@@ -233,10 +177,8 @@ the **Migration History**. It includes all successful and failed migrations of t
 
 ## Architecture
 
-Lift has 2 parts: a **Lift Client** and a **Lift Engine**. By default, the Lift Engine runs locally as a
-[sidecar process](https://blog.davemdavis.net/2018/03/13/the-sidecar-pattern/). This makes it easy to get started. As your team's data requirements become more
-complex, you may prefer to handle your migrations on a remote host where you have fine-grained access control over migrations. We will make this possible in the
-future.
+Lift has 2 parts: a **Lift Client** and a **Lift Engine**. The Lift Engine runs locally as a
+[sidecar process](https://blog.davemdavis.net/2018/03/13/the-sidecar-pattern/) to the Lift Client. 
 
 ```
```

Comment:
Can we make this diagram align a bit better and have straight lines @matthewmueller?

Reply:
You mean center-aligned?

Which lines aren't straight, the arrows? Or are you talking about the line segments not being connected?